### DISCUSSION TITLE
How to open a new FreeTube window from the freetube command line ?

### DISCUSSION THREAD

**Discussion by sebma** [FreeTubeApp/FreeTube#2100] [Q&A] [answered]
Hi everyone,

How to open a new windows from the freetube command line ?

I tried to find the freetube command by typing : `freetube --help` but it does not show anything on the terminal.

**Comment by PrestonN** [comment, marked as answer]
At this time, FreeTube isn't much of a CLI program and doesn't really provide any CLI flags to toggle. You can start the program with the `freetube` command but that's about all it does at this time. You can also paste a YouTube URL after the `freetube` command to open that URL into the app. Adding a CLI menu in the future may be something that we can work on someday, but it currently isn't a priority.

**Reply by EllieMelloy** [reply]
Please consider adding this capability, it's frustrating when wanting to start a new instance of freetube with a kb shortcut but it just switches back to an already open instances.

**Reply by absidue** [reply]
@RodneyRabbit You can already start a new window with <kbd>Ctrl</kbd>+<kbd>N</kbd> or <kbd>Command</kbd>+<kbd>N</kbd>.

https://docs.freetubeapp.io/usage/keyboard-shortcuts/

**Reply by EllieMelloy** [reply]
Understood, I know about Ctrl-N but I need to use autohotkey for global shortcuts.

It probably sounds trivial but I have a restricting hand disability. Moving around the keyboard takes much effort and time. I get that it's a very unique individual problem though and might not benefit many people. I'm sure I'll find another solution.

I tried to create a shortcut to the exe but it just activates an existing instance with apparently no logic, meaning I then have to use Ctrl-N, Alt-D, paste yt link or enter search, then go back and minimise the previous instance to not mess up my careful window placement to work with my condition. 

It's fine with other apps e.g. Firefox - I can assign a hotkey Ctrl-Alt-F which just starts a new instance without messing up any existing window placement. The thing with autohotkey is it works well when launching a single application, but trying to automate it with window classes and titles, or heaven forbid a load of unnecessary Alt-Tab and other key combos, it behaves inconsistent.

**Comment by anonaddict** [comment]
CLI would be nice though I can see why it wouldn't be a priority for many

**Reply by EllieMelloy** [reply]
Would love this capability so I can assign an autohotkey shortcut to start a new freetube instance, like can be done for every other application. Currently it just switches back to an already open window which is strange.

**Reply by Gevaarlijk** [reply]
Yes please!
